test/tools/config_load_check/config_load_check.cc

```diff
@@ -19,6 +22,11 @@ int main(int argc, char* argv[]) {
     return EXIT_FAILURE;
   }
   try {
+
+    Envoy::Event::Libevent::Global::initialize();
```

Comment:
Was it the switch to `ValidationClusterManagerFactory` that caused the absent libevent to be noticed? I thought we had an ASSERT to validate that libevent was up somewhere on the init path, so curious why this wasn't firing before.

Reply:
Yes, the switch to `ValidationClusterManagerFactory` that highlighted that this too was missing.